bin/des-pizza-cutter-fit-gaia-mask

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Comment:
Is this script meant to be a general command-line tool or a one-off script? If it is a one-off, we should move it to a top-level directory like `scripts` or something.

Reply:
its a one off